README.md

```diff
@@ -1,6 +1,6 @@
 ## Hi, I am Kaiwalya! Great to see you here! <img src="https://raw.githubusercontent.com/aemmadi/aemmadi/master/wave.gif" width="30px">
 
-I am Kaiwalya Koparkar, currently pursuing my Diploma majoring in computer science from [M. S. Gosavi Polytechnic Institute](https://gespoly.org/). I am a tech enthusiast & an open-source advocate. I am always open to collaborating on projects and innovative/disruptive ideas. Find out more about me & feel free to connect with me here:
+I am Kaiwalya Koparkar, Currently I am working as Community Manager Intern at Sawo-Labs Community and Part time Program manager at Community Classroom. I am a Full Stack Developer with experience building open source projects. I am a tech enthusiast & an open-source advocate. I am always open to collaborating on projects and innovative/disruptive ideas. Find out more about me & feel free to connect with me here:
```

Comment:
Please remove `Part Time` part from the community classroom. Also please add GCE, mentor at MLH info

Reply:
okay @kaiwalyakoparkar

Reply:
chekout now